src/time/format.cr

```diff
@@ -49,12 +49,21 @@
 # * **%:z**: time zone as hour and minute offset from UTC with a colon (+09:00)
 # * **%::z**: time zone as hour, minute and second offset from UTC with a colon (+09:00:00)
 struct Time::Format
-  # The ISO 8601 date format. This is just `"%F"`.
+  # The [ISO 8601](http://xml.coverpages.org/ISO-FDIS-8601.pdf) date format. This is just `"%F"`.
   ISO_8601_DATE = new "%F"
 
-  # The ISO 8601 datetime format. This is just `"%FT%X%z"`.
   ISO_8601_DATE_TIME = new "%FT%X%z"
 
+  # The [RFC 3339](https://tools.ietf.org/html/rfc3339)/[ISO 8601](http://xml.coverpages.org/ISO-FDIS-8601.pdf) datetime format.
+  # This is just `"%FT%X%z"`.
+  RFC_3339 = new "%FT%X%:z"
+
+  # The [RFC 1123](https://tools.ietf.org/html/rfc1123#page-55) datetime format.
+  RFC_1123 = new "%a, %d %b %Y %H:%M:%S GMT", Time::Kind::Utc
+
+  # The [RFC 2822](https://tools.ietf.org/html/rfc2822) datetime format.
+  RFC_2822 = new "%a, %d %b %Y %H:%M:%S %z"
```

Comment:
You're right. Keeping track of all these formats is really disturbing... So we should keep RFC 2822 and add notes, that it can also be used (mostly) for 822 and 1123?